hw/ip_templates/rv_plic/fpv/tb/coverage_waivers.tcl.tpl

```diff
@@ -18,3 +18,14 @@ check_cov -waiver -add -start_line 25 -end_line 56 -type {statement} -instance\
 # The port data_o in u_data_chk is untied and used nowhere.
 check_cov -waiver -add -start_line 25 -end_line 81 -type {statement} -instance\
  {dut.u_reg.u_chk.u_chk} -comment {data_o is untied}
+
+# These are bunch of software controlled registers with RO and RW Software access. For RO
```

Comment:
Thinking about it, none of the registers in `rv_plic` have hardware writes. As a result, the `ds` output port on `prim_subreg` never gets connected. I think the comments can be shorter as a result (we don't really care about whether the code is dead or not since we will never look at the `ds` signal anyway).

To be super paranoid, maybe we should chuck in an assertion that says `de` is always zero (which means there is never a write from hardware) ?